CMakeLists.txt

```diff
@@ -1,41 +1,28 @@
-if(CCPP)
-
-  if(DEBUG)
-    set(_ccpp_debug_arg "--debug")
-  endif()
-  if(DEFINED CCPP_SUITES)
-    set(_ccpp_suites_arg "--suites=${CCPP_SUITES}")
-    message("Calling CCPP code generator (ccpp_prebuild.py) for suites ${_ccpp_suites_arg} ...")
-  else()
-    message("Calling CCPP code generator (ccpp_prebuild.py) for all available suites ...")
-  endif()
-  execute_process(COMMAND ${Python_EXECUTABLE}
-                          "ccpp/framework/scripts/ccpp_prebuild.py"
-                          "--config=ccpp/config/ccpp_prebuild_config.py"
-                          "--builddir=${CMAKE_CURRENT_BINARY_DIR}" ${_ccpp_suites_arg} ${_ccpp_debug_arg}
-                  WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-                  OUTPUT_FILE ${CMAKE_CURRENT_BINARY_DIR}/ccpp_prebuild.out
-                  ERROR_FILE  ${CMAKE_CURRENT_BINARY_DIR}/ccpp_prebuild.err
-                  RESULT_VARIABLE RC)
-  # Check return code from ccpp_prebuild.py
-  if(NOT RC EQUAL 0)
-    message(FATAL_ERROR "An error occured while running ccpp_prebuild.py, check ${CMAKE_CURRENT_BINARY_DIR}/ccpp_prebuild.{out,err}")
-  endif()
-  # this should not be necessary; including CCPP_*.cmake here and passing
```

Comment:
Happy to see this is not needed anymore.

Reply:
Turns out that a small change in ccpp-physics (a bugfix, actually) was all that was required.